proto/prysm/v1alpha1/wrapper/beacon_block.go

```diff
@@ -15,6 +17,21 @@ var (
 	_ = block.BeaconBlockBody(&altairBeaconBlockBody{})
 )
 
+// WrappedSignedBeaconBlock will wrap a signed beacon block to conform to the
+// signed beacon block interface.
+func WrappedSignedBeaconBlock(i interface{}) (block.SignedBeaconBlock, error) {
```

Comment:
Nice! Can we also add a unit test to it?

Reply:
Good idea, thanks!